core/src/callback.jl

```diff
@@ -379,10 +405,17 @@ end
 
 "Copy the current flow to the SavedValues"
 function save_flow(u, t, integrator)
-    vcat(
-        get_tmp(integrator.p.graph[].flow_vertical, 0.0),
-        get_tmp(integrator.p.graph[].flow, 0.0),
-    )
+    (; graph) = integrator.p
+    (; flow_integrated, flow_vertical_integrated, tprev_flow_save) = graph[]
+    Δt = t - tprev_flow_save[1]
```

Comment:
I don't like the length 1 vector `tprev_flow_save` very much, but we still need something mutable like this because if all timesteps are saved the interval between saves is not constant. Or is there a way to obtain this time interval in a different way?

Reply:
Perhaps if we save all timesteps we use `integrator.tprev`, and otherwise the dt is constant.